test/users/engin/refComparison.skipif

```diff
@@ -0,0 +1 @@
+CHPL_COMM!=none
```

Comment:
Should it really be "CHPL_COMM==none" , which would only run the test in multilocale configurations?